fdbt-dev/data/netexData/returnCircularGroup.xml

```diff
@@ -88,7 +90,10 @@
             <ValueSet version="1.0" id="Branding" classOfValues="Branding">
               <Name>Operator Branding</Name>
               <values>
-                <Branding version="1.0" id="op:WBTR@brand"/>
+                <Branding version="1.0" id="op:WBTR@brand">
+                  <Name>Warrington's Own Buses</Name>
+                  <Url>http://www.blackpooltransport.com</Url>
```

Comment:
Ah yeah good spot - the name comes from the matching JSON but the URL comes from the hard coded unit test data for operators (instead of from the DB) so I think it's probably fine?